data/migrations/versions/506f29126e2b_increase_repomirror_s_upstreamregistry_.py

```diff
@@ -0,0 +1,36 @@
+"""increase repomirror's upstreamregistry password length
+
+Revision ID: 506f29126e2b
+Revises: 
+Create Date: 2024-07-04 17:53:10.170420
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '506f29126e2b'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
```

Comment:
You can get rid of these lines and just specify a down revision here, it should be the latest version in alembic's history (946f0e90f9c9)